oshi-core/src/main/java/oshi/software/os/linux/LinuxOperatingSystem.java

```diff
@@ -712,4 +715,26 @@ public static long getHz() {
         return USER_HZ;
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public OSService[] getServices() {
+        //Sort by PID
+        OSProcess[] process =  getChildProcesses(1, 0, OperatingSystem.ProcessSort.PID);
+        File etc = new File("/etc/init");
+        File[] files = etc.listFiles();
```

Comment:
Need a null check on `etc` here (I know `/etc/init` should always exist, but that shouldn't crash OSHI if it doesn't.)  Same issue on the FreeBSD and Solaris implementations.